test/opset_pass/product_opset_pass.cpp

```diff
@@ -19,13 +19,14 @@
 
 #include "ngraph/ngraph.hpp"
 #include "ngraph/pass/manager.hpp"
+#include "ngraph/pass/opset1_downgrade.hpp"
 #include "ngraph/pass/opset1_upgrade.hpp"
 #include "util/type_prop.hpp"
 
 using namespace std;
 using namespace ngraph;
 
-TEST(serialize, opset1_product_upgrade)
+TEST(product, opset1_product_upgrade)
```

Comment:
I don't force it but this name is better IMHO.
```suggestion
TEST(opset_downgrade, opset1_product_upgrade)
```